lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
+import PropTypes  from "prop-types";
 
 const Tab = props => {
-  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
-      if they match, the className should be: 'tab active-tab', 
+  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop,
+      if they match, the className should be: 'tab active-tab',
       if it is not it should just be 'tab'*/
+       if(props.tab === props.mySelectedTab) {
   return (
     <div
-      className={''}
+      className={'tab active-tab'}
       onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
+        /* Replace this dummy click handler function with your selectTabHandler function from props
          you'll need to pass the `tab` in as an argument to this handler. */
+         props.myTabHandler(props.tab)
       }}
     >
       {props.tab.toUpperCase()}
     </div>
   );
+} else {
+        return (
+            <div
+                className={'tab'}
+                onClick={() => {
+                    /* Replace this dummy click handler function with your selectTabHandler function from props
+                     you'll need to pass the `tab` in as an argument to this handler. */
+                    props.myTabHandler(props.tab)
+                }}
+            >
+                {props.tab.toUpperCase()}
+            </div>
+        );
+    }
```

Comment:
You could save a lot of copied code here by just don't the conditional check inside the className like this ```className={`tab ${props.tab === props.selectedTab? "active-tab" : ""} `} ```